src/main/java/com/example/gradecalculator/service/UserService.java

```diff
@@ -20,7 +20,8 @@ public class UserService {
     private final UserRegistrationMapper userRegistrationMapper;
 
     @Autowired
-    public UserService(UserRepository userRepository, UserTypeRepository userTypeRepository, PasswordEncoder passwordEncoder, UserRegistrationMapper userRegistrationMapper) {
+    public UserService(UserRepository userRepository, UserTypeRepository userTypeRepository
+            , PasswordEncoder passwordEncoder, UserRegistrationMapper userRegistrationMapper) {
```

Comment:
```suggestion
    public UserService(UserRepository userRepository, 
                       UserTypeRepository userTypeRepository,
                       PasswordEncoder passwordEncoder, 
                       UserRegistrationMapper userRegistrationMapper) {
```
not a fan of the current formatting